packages/web/src/components/data-entry/FormTile.js

```diff
@@ -59,6 +66,69 @@ const Divider = (props) => {
   )
 }
 
+const TrackAvailabilityButton = (props) => {
```

Comment:
Can we move this to a separate TS file so we're not adding any more JS?

Reply:
this is meant to be removed after launch, only there for the feature flag so seems to me that we don't need to separate it. lmk if you feel strongly

Reply:
that's fine if just feature flagged. but let's add a comment tracking linear issue then here